UI/Pages/Game.razor

```diff
@@ -0,0 +1,11 @@
+@page "/game";
+@using TicTacGo.Enums
+
+<div class="bg-gradient-to-r from-violet-900 to-purple-900 h-screen">
+    <GameGrid GridCells="@GridCells" ActiveMark = "@ActiveMark"/>
```

Comment:
Remove extra spaces around the `ActiveMark`. Before and after the equal sign. And add a space before the `/>`.